tests/unit/lms/services/canvas_api/_authenticated_test.py

```diff
@@ -46,30 +47,32 @@ def test_send_refreshes_and_retries_for_ProxyAPIAccessTokenError(
             "success",  # Then finally our successful content request
         )
 
-        call_args = ("METHOD", "/path", sentinel.schema, sentinel.params)
+        call_args = ("METHOD", "/path", sentinel.schema, (10, 10), sentinel.params)
         result = authenticated_client.send(*call_args)
 
-        basic_client.send.assert_has_calls(
-            (
-                # Initial call with access token
-                call(
-                    *call_args,
-                    headers={"Authorization": f"Bearer {oauth_token.access_token}"},
+        assert basic_client.send.call_args_list == [
```

Comment:
I'm not sure why this test was using `assert_has_calls()`, `call_args_list == ...` is the correct thing to use here